### ISSUE TITLE
case-sensitive autocompletion

### ISSUE THREAD

**Issue by gvko** [warpdotdev/Warp#755]
### Describe the solution you'd like?

Eg. when I write `cd de` and click `Tab` I don’t want to see the options `developer` and `Desktop` and to have to choose from them in the dropdown. I want to get `developer` autocompleted. And if I write `cd De` I want `Desktop` autocompleted.
(see attached screenshot)

### Is your feature request related to a problem? Please describe.

This is frustrating, since I gotta choose between options in the pop-up menu instead of direct autocompletion. I know this sounds kind of entitled 😄 , sorry for that. Just expressing desire for a feature that I currently have in iTerm. :)

### Additional context

![photo_2022-02-17 13 23 15](https://user-images.githubusercontent.com/1754630/154490752-4aab3063-a596-4084-82e5-cb7788feb52e.jpeg)

**Comment by apeace** [comment]
I think the existing behavior might be a better default. But for me, I would like a setting to change it back to case-sensitive because I am so used to working that way.

**Comment by gvko** [comment]
> I think the existing behavior might be a better default. But for me, I would like a setting to change it back to case-sensitive because I am so used to working that way.

Yes, @apeace , actually iTerm has a setting where you can turn on/off the case-sensitive autocompletion. So, I think a setting like that would be awesome, fitting everyone's needs.
And I think it's quite a crucial setting, since we use file/dir autocompletion tens of times a day in the terminal.
Right now this is pretty much the only thing that's stopping me from moving to Warp as my default terminal.

**Comment by elviskahoro** [comment]
Thanks for opening will update you when this is a work in progress!
Marked high priority btw!

**Comment by Virviil** [comment]
Any updates regarding this issue?

For me it's critical one

**Comment by warpdotdev-dx** [comment]
No updates, unfortunately. :(
